Assets/MRTK/Examples/packagetemplate.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "com.microsoft.mixedreality.toolkit.examples",
     "displayName": "Mixed Reality Toolkit Examples",
+    "featureCategory": "Mixed Reality Toolkit",
```

Comment:
Do we want to append this tag with `ms` or `msft` or something, or do you think the chances of this colliding with anything npm or Unity might add is pretty low?

Reply:
that's not a bad thought! thx!

Reply:
done